actions/run-stage/action.yaml

```diff
@@ -5,9 +5,6 @@ inputs:
   project:
     description: The project to run
     required: true
-  stage:
```

Comment:
I guess we should also rename this action, as it no longer "runs a stage". Maybe `actions/generate-kubernetes-manifests` ?